jekyll/_cci2/configuration-reference.adoc

```diff
@@ -1846,7 +1846,7 @@ jobs:
 [NOTE]
 ====
 * `setup_remote_docker` is not compatible with the `machine` executor. See xref:docker-layer-caching#machine-executor[Docker Layer Caching in Machine Executor] for information on how to enable DLC with the `machine` executor.
-* The `version` key is not currently supported on CircleCI server. Contact your system administrator for information about the Docker version installed in your remote Docker environment.
+* The `version` key is not currently supported on CircleCI server. Contact your system administrator for information about the Docker version installed in your remote Docker environment. If you are on server 4.x, you can find the default AWS AMI xref:listserver/v4.2/operator/manage-virtual-machines-with-vm-service/#default-aws-ami-lists[here]
```

Comment:
```suggestion
* The `version` key is not currently supported on CircleCI server. Contact your system administrator for information about the Docker version installed in your remote Docker environment. If you are on server 4.x, you can find the default AWS AMI xref:server/latest/operator/manage-virtual-machines-with-vm-service/#default-aws-ami-lists[here].
```

Reply:
If we change to latest here once 4.6 docs are live this link will always point to most recent server docs. Thank you!